Rubberduck.Parsing/Annotations/MemberAttributeAnnotation.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Rubberduck.Parsing.Grammar;
+using Rubberduck.VBEditor;
+
+namespace Rubberduck.Parsing.Annotations
+{
+    public class MemberAttributeAnnotation : AttributeAnnotationBase
+    {
+        public MemberAttributeAnnotation(QualifiedSelection qualifiedSelection, VBAParser.AnnotationContext context, IReadOnlyList<string> parameters)
+        :base(AnnotationType.MemberAttribute, qualifiedSelection, context, parameters.Skip(1).ToList())
```

Comment:
Ditto the null check here. `parameters?.Skip(1).ToList() ?? Enumerable.Empty<string>()`